filebeat/harvester/config.go

```diff
@@ -45,6 +46,7 @@ type harvesterConfig struct {
 	CloseRemoved         bool                       `config:"close_removed"`
 	CloseRenamed         bool                       `config:"close_renamed"`
 	CloseEOF             bool                       `config:"close_eof"`
+	CloseTimeout         time.Duration              `config:"close_timeout" validate:"min=0,nonzero"`
```

Comment:
I agree that nonzero should be removed. But -1 doesn't make sense from my point of view?